package.json

```diff
@@ -30,7 +30,9 @@
     "dotenv": "^4.0.0",
     "json-loader": "latest",
     "jsonfile": "^3.0.1",
+    "mem": "^4.0.0",
```

Comment:
Also, why was this added?

Reply:
It's also a package with a vulnerability

Reply:
Ah sorry, it's a new one, I missed that. It seems that the original `1.1.0` version is still hanging around:

https://github.com/execonline-inc/vimeo-upload-lambda/pull/8/files#diff-8ee2343978836a779dc9f8d6b794c3b2R25998

This could be trickier to fix since it isn't just a patch version difference.

Reply:
I rebuilt yarn.lock.

Reply:
Okay, now it looks nothing is using `mem` anymore in `yarn.lock`, so I think it can be removed as a dependency.

Reply:
Updated @pjrebsch